example/react-fundamentals/src/setupTests.js

```diff
@@ -1 +1,6 @@
 import '@kentcdodds/react-workshop-app/setup-tests'
+import {server} from './test/server'
+
+beforeAll(() => server.listen())
```

Comment:
Let's set onUnhandledRequest as "error" here.

Reply:
I have added it, on both node and browser.